components/director/pkg/sanitize/directive.go

```diff
@@ -0,0 +1,45 @@
+package sanitize
```

Comment:
this is actually hasScopes, just with different "error outcome" - in this case nil. Maybe move all these things in has scopes (this comment builds ontop of the one about using the same directive struct). Probably in regards to the tests you would have to add one or two extra tests to the existing scope directive tests and avoid some copying there as well